starlark/src/stdlib/mod.rs

```diff
@@ -956,6 +957,7 @@ pub fn global_environment() -> Environment {
 pub fn global_environment_with_extensions() -> Environment {
     let env = global_environment();
     let env = structs::global(env);
+    let env = inspect::global(env);
```

Comment:
Same as the other PR: consider not adding it to the global env but rather to repl and test envs.